example/sofast_fixed/example_measurement_from_image.py

```diff
@@ -0,0 +1,83 @@
+from os.path import join, dirname
+
+import cv2 as cv
+
+import opencsp.app.sofast.lib.image_processing as ip
+from opencsp.app.sofast.lib.MeasurementSofastFixed import MeasurementSofastFixed
+from opencsp.common.lib.geometry.Vxyz import Vxyz
+from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
+import opencsp.common.lib.render.figure_management as fm
+import opencsp.common.lib.render_control.RenderControlAxis as rca
+import opencsp.common.lib.render_control.RenderControlFigure as rcfg
+import opencsp.common.lib.tool.file_tools as ft
+import opencsp.common.lib.tool.log_tools as lt
+
+
+def example_create_measurement_file_from_image():
+    """Example that creates a SofastFixed measurement file from an image. The image has
+    a point LED light near origin dot.
+    1. Load image
+    2. Define measurement parameters
+    3. Find location of origin point
+    4. Create measurement object
+    5. Save measurement as HDF5 file
+    6. Plot image with annotated origin dot
+    """
+    # General setup
+    # =============
+    dir_save = join(dirname(__file__), 'data/output/measurement_file')
+    ft.create_directories_if_necessary(dir_save)
+    lt.logger(join(dir_save, 'log.txt'), lt.log.INFO)
+
+    # 1. Load image
+    # =============
+    file_meas = join(opencsp_code_dir(), 'test/data/sofast_fixed/data_measurement/measurement_facet.h5')
+    measurement_old = MeasurementSofastFixed.load_from_hdf(file_meas)
+    image = measurement_old.image
+
+    # 2. Define measurement parameters
+    # ================================
+    v_measure_point_facet = Vxyz((0, 0, 0))  # meters
+    dist_optic_screen = 10.008  # meters
+    name = 'NSTTF Facet'
+
+    # 3. Find location of origin point
+    # ================================
+
+    # Find point light
+    params = cv.SimpleBlobDetector_Params()
```

Comment:
Idea time! It would be nice if you could retrieve these parameters from image_processing with some values such as `filterByCircularity` set for you:

```python
params = ip.dot_target_blob_params(minDistBetweenBlobs=2, minArea=3, maxArea=30)

def dot_target_blob_params(
    minDistBetweenBlobs=2,
    filterByArea=True,
    minArea=3,
    maxArea=30,
    filterByCircularity=True,
    minCircularity=0.8,
    filterByConvexity=False,
    filterByInertia=False,
):
    params = cv.SimpleBlobDetector_Params()
    params.minDistBetweenBlobs = minDistBetweenBlobs
    params.filterByArea = filterByArea
    params.minArea = minArea
    params.maxArea = maxArea
    params.filterByCircularity = filterByCircularity
    params.minCircularity = minCircularity
    params.filterByConvexity = filterByConvexity
    params.filterByInertia = filterByInertia
    return params
```

To be clear, not necessary for this PR, and not necessary generally. Just an idea.

Reply:
This is an interesting idea! There are other inputs that SimpleBlobDetector_Params takes that I haven't typically used. Perhaps have a `default_blob_detector_params()` function or something? I'll have to think about how best to implement it, but I do really like it. Will probably wait until a future PR.